README.md

```diff
@@ -300,6 +308,24 @@ $ sfdx sgd:source:delta --from commit --ignore-destructive destructiveignore
 
 Note that in a situation where only the `--ignore [-i]` parameter is specified (and `--ignore-destructive [-D]` is not specified), then the plugin will ignore items matching `--ignore [-i]` parameter in all situations: Addition, Modification and Deletion.
 
+### Explicitly including specific files for inclusion or destruction regardless of diff:
```

Comment:
**praises:** nice, you even thought about explaining the new feature in the README 👏

Reply:
👍